pkg/scheduler/metrics/metrics.go

```diff
@@ -204,6 +206,23 @@ func InitMetrics() {
 			Help:           "Number of running goroutines split by the work they do such as binding.",
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"operation"})
+	GoroutinesDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      SchedulerSubsystem,
+			Name:           "goroutines_duration_seconds",
+			Help:           "how long each preemption goroutine takes to complete",
```

Comment:
I think this metric is not just for preemption, since `Parallelizer.Until` is used in other phases as well.
Also we better keep the style consistent with other help messages in this file.

```
"Duration in seconds for running goroutines split by the work they do such as binding."
```